apps/server/src/tests/auth.test.ts

```diff
@@ -0,0 +1,78 @@
+import { client, db } from '../clients/postgres';
+import { Users } from '../db/schema';
+import { httpServer } from '../index';
+import request from 'supertest';
+import { eq } from 'drizzle-orm';
+import { generateEncodedString } from '../utilities/generateEncodedString';
+
+const { JWT_AUTH_SECRET_KEY, JWT_AUTH_EXPIRES_IN } = process.env;
+
+const server = request(httpServer);
+
+interface IUserRegisterObject {
+    fullName: string;
+    email: string;
+    password: string;
+    passwordConfirm: string;
+}
+
+const userRegisterObject: IUserRegisterObject = {
+    fullName: 'Hasan toker',
+    email: 'tokerto7001@gmail.com',
+    password: '12345678',
+    passwordConfirm: '12345678',
+};
+
+describe('Authentication process test', () => {
+    describe('Register', () => {
```

Comment:
While creating your test suits, please be careful with the 3A s of testing. Read more about it from here [https://thinkster.io/tutorials/blogs/the-aaa-structure-of-tests](https://thinkster.io/tutorials/blogs/the-aaa-structure-of-tests)